docs/globalnet.md

```diff
@@ -1,66 +1 @@
-
-### IP Address Manager (IPAM)
-
-The IP Address Manager (IPAM) component does the following:
-
-* Creates a pool of IP addresses based on `GlobalCIDR` configured on cluster.
-* On creation of a Pod/Service, allocates a GlobalIp from the GlobalIp pool.
-* Annotates the Pod/Service with `submariner.io/GlobalIp=<global-ip>`.
-* On deletion of a Pod/Service, releases its GlobalIp back to the pool.
-
-### Globalnet
-
-This component is responsible for programming the routing entries and IPTable rules and does the following:
-
-* Creates initial IPTables chains for Globalnet rules.
-* Whenever a pod is annotated with Global IP, creates an egress SNAT rule to convert source ip from PodIp to pod's GlobalIp on the Gateway Node.
-* Whenever a service is annotated with Global IP, creates an ingress rule to direct all traffic destined to service's GlobalIp to the service's `kube-proxy` IPTables chain which in turn directs traffic to service's backend pods.
-* On deletion of pod/service, clean up the rules from the gateway node.
-
-Globalnet currently relies on `kube-proxy` and thus will only work with deployments that use `kube-proxy`.
-
-## Service Discovery - Lighthouse
-
-Connectivity is only part of the solution as pods still need to know the IPs of services on remote clusters.
-
-This is achieved by enhancing [lighthouse](https://github.com/submariner-io/lighthouse) with support for GlobalNet. The Lighthouse controller adds the service's GlobalIp to the `MultiClusterService` object that is distributed to all clusters. The [lighthouse plugin](https://github.com/submariner-io/lighthouse/tree/master/plugin/lighthouse) then uses the service's GlobalIp when replying to DNS queries for the service.
-
-# Installation
-
-TBD.
-
-# Testing
-
-Refer [testing guide](testing.md) for instructions on how to test, same steps apply.
-
-# Known Issues
-
-# Building
-
-Nothing extra needs to be done to build `submariner-globalnet` as it is built with the standard submariner build.
-
-# TODO
-
-* [Host Network to remote Service Connectivity](https://github.com/submariner-io/submariner/issues/464)
-* [Provision to specify globalnet-cluster-size during subctl join](https://github.com/submariner-io/submariner-operator/issues/268)
-* [Globalnet Controller support for non IPTables based kubeproxy](https://github.com/submariner-io/submariner/issues/382)
\ No newline at end of file
+[Link to Globalnet Architecture](https://submariner-io.github.io/architecture/globalnet/)
```

Comment:
We can remove this document now, as the reference to this document does not exist anymore.

Reply:
Done